contrib/completion/zsh/_docker-compose

```diff
@@ -19,52 +19,49 @@
 #  * @felixr docker zsh completion script : https://github.com/felixr/docker-zsh-completion
 # -------------------------------------------------------------------------
 
-# For compatibility reasons, Compose and therefore its completion supports several
-# stack compositon files as listed here, in descending priority.
-# Support for these filenames might be dropped in some future version.
-__docker-compose_compose_file() {
-    local file
-    for file in docker-compose.y{,a}ml ; do
-        [ -e $file ] && {
-            echo $file
-            return
-        }
-    done
-    echo docker-compose.yml
+__docker-compose_q() {
+    docker-compose 2>/dev/null $compose_options "$@"
 }
 
 # Extracts all service names from docker-compose.yml.
```

Comment:
@sdurrheimer not sure what the change is…

Maybe I misunderstood something, but I think @albers asked me to update that comment rather than removing it.

I then pointed out that I removed it because it was no longer needed, and there is no corresponding comment on the same function in the bash script.

I take what @albers wrote after to mean that after some deliberation, he was okay with the comment being removed.

Again, I might be wrong here, and I'd gladly add a comment if there's a need for one :)